include/mata/nfa/nfa.hh

```diff
@@ -601,24 +588,6 @@ Nfa somewhat_simple_revert(const Nfa& aut);
 // Removing epsilon transitions
 Nfa remove_epsilon(const Nfa& aut, Symbol epsilon = EPSILON);
```

Comment:
sounds like in place, but don't know, epsilon_free, :(

Reply:
`epsilon_removal()`? `epsilon_removed()`?